src/app.js

```diff
@@ -1,21 +1,46 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const { getContentType } = require('./getContentType');
+
+const PORT = process.env.PORT || 8800;
+
+const server = http.createServer((req, res) => {
+  const normilizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const partOfPath = normilizedURL.pathname.slice(1).split('/');
+
+  if (partOfPath[0] !== 'file') {
+    res.end('Path must start with /file');
+
+    return;
+  }
+
+  const fileName = partOfPath.slice(1).join('/') || 'index.html';
+
+  const absolutePath = path.join(
+    __dirname, `public/${fileName}`,
+  );
+
+  fs.readFile(absolutePath, 'utf-8', (err, data) => {
+    if (err) {
+      res.statusCode = 404;
+      res.end('File not found');
+
+      return;
+    }
+
+    const extension = path.extname(absolutePath);
+    const contentType = getContentType(extension);
+
+    res.getHeader('Content-type', contentType);
```

Comment:
The method 'getHeader' is non-existent on the 'res' object. It should be changed to 'setHeader' to set the content type of the response.